src/main/java/com/manardenza/servise/HotelService.java

```diff
@@ -0,0 +1,46 @@
+package com.manardenza.servise;
+
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.Hotel;
+
+import java.util.List;
+import java.util.Map;
+
+public final class HotelService {
+
+    private static HotelService instance;
+    private HotelDaoImpl hotelDao;
+
+    private HotelService() {
+        this.hotelDao = HotelDaoImpl.getInstance();
+    }
+
+    public static HotelService getInstance() {
+        if (instance == null) {
+            instance = new HotelService();
+        }
+        return instance;
+    }
+
+    public List<Hotel> findHotelByName(String hotelName) {
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)